compiler/src/dotty/tools/dotc/config/Feature.scala

```diff
@@ -97,4 +97,22 @@ object Feature:
     else
       false
 
+  private val assumeExperimentalIn = Set("dotty.tools.vulpix.ParallelTesting")
+
+  def checkExperimentalFeature(which: String, srcPos: SrcPos = NoSourcePosition)(using Context) =
+    def hasSpecialPermission =
+      new Exception().getStackTrace.exists(elem =>
+        assumeExperimentalIn.exists(elem.getClassName().startsWith(_)))
```

Comment:
Why would we need this?

Reply:
So that we can run the tests from a release compiler

Reply:
Why would we want to run tests on experimental features on a release compiler? These cannot be used and there will be a nightly that will cover those tests on that day anyway.

Reply:
The tests are there so that we can make sure that the compiler *code base* can in fact handle the experimental features. There's only one codebase. Those tests can be run from either release or experimental versions. The fact that release compilers flag experimental imports as errors unless the _version string_ is a  nightly or snapshot is orthogonal to this.

Reply:
Note that the previous workaround of `experimental` subdirectores has been reverted.